pynicotine/shares.py

```diff
@@ -477,11 +477,11 @@ def is_hidden(folder, filename=None, entry=None):
         """Stop sharing any dot/hidden folders/files."""
 
         # If the last folder in the path starts with a dot, or is a Synology extended
-        # attribute folder, we exclude it
+        # attribute, snapshot or recycle bin folder, we exclude it
         if filename is None:
             last_folder = os.path.basename(folder)
 
-            if last_folder.startswith(".") or last_folder == "@eaDir":
+            if last_folder.startswith(".") or last_folder == ["@eaDir", "#recycle", "#snapshot"]:
```

Comment:
Condition is equal to entire list. You need `in` instead of `==`.

Reply:
Thank you, fixed.